src/app/helpers/page/pageInfo.ts

```diff
@@ -41,17 +42,12 @@ export class PageInfo implements IPageInfo {
     this.resolvers = args.resolvers ?? {};
   }
 
-  /**
-   * Set Menu Route for PageInfo. This also modifies the menu route to include
-   * the associated target component.
-   */
-  public setMenuRoute(target: Type<PageComponent>, menu: MenuRoute) {
-    this.pageRoute = menu;
+  public setComponent(target: Type<PageComponent>): void {
     this.component = target;
     this.route.pageComponent = target;
```

Comment:
might be good to add some validation - `menuRoute` is optional in the interface which means it could be omitted. That would be an error?

Reply:
Validation in the caller (`linkToRoute`) might also be acceptable